### PR TITLE
Add EasyCaching.LiteDB

### PR DESCRIPTION
**Pull request by maikebing**
LiteDB is a serverless database delivered in a single small DLL (< 450kb) fully written in .NET C# managed code (compatible with .NET 4.5 and NETStandard 2.0).

Portable UWP and Xamarin iOS/Android
ACID transactions
Single datafile (like SQLite)
Recovery data in writing failure (WAL mode)
Map your POCO class to BsonDocument
Fluent API for custom mapping
Cross collections references (DbRef)
Store files and stream data (like GridFS in MongoDB)
LINQ query support
FREE for everyone - including commercial use

#173

### CODE REVIEW

File: src/EasyCaching.LiteDB/EasyCaching.LiteDB.csproj [hunk 1]
```diff
@@ -0,0 +1,39 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="../../build/version.props" />
+  <Import Project="../../build/releasenotes.props" />
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Owners>maikebing</Owners>
+    <Authors>maikebing</Authors>
+    <Version>1.0.1</Version>
```
Comment (catcherwong) [inline src/EasyCaching.LiteDB/EasyCaching.LiteDB.csproj:8]: Can we read this version with `$(EasyCachingXXXPackageVersion)`  from `version.props`?

**Review by catcherwong — APPROVED** [review]
LGTM.

File: src/EasyCaching.LiteDB/EasyCaching.LiteDB.csproj (hunk 1 above)
Comment (maikebing) [reply src/EasyCaching.LiteDB/EasyCaching.LiteDB.csproj]: Okay, I've pushed the fix.

**Comment by catcherwong** [comment]
could you please change the target branch from `master` to `dev`?

**Review by catcherwong — APPROVED** [review]

**Pull request merged**